### ISSUE TITLE
Export multiple Charts to multi page single pdf

### ISSUE THREAD

**Issue by mayurtanna** [highcharts/highcharts#5834]
#### Expected behaviour

I want to export 16 charts to multi page single pdf. But all charts are coming on a single page. Is there any way we can have page breaks ? We are using  code like this: https://github.com/highcharts/highcharts/blob/master/samples/highcharts/exporting/multiple-charts/demo.js
#### Actual behaviour

All charts gets printed on a single long page which is not print friendly.
#### Live demo with steps to reproduce

http://jsfiddle.net/highcharts/Ldsmk030/

**Comment by TorsteinHonsi** [comment]
Thanks for reporting! Our export server is made for creating one page per SVG, but we'll see what we can do.

**Comment by mayurtanna** [comment]
Thank you for the comment. If we can get a multiple page PDF by passing multiple SVGs to the export server, that would also work. I think this is one of the important feature which is required. Thank you.

**Comment by binarudani** [comment]
Hi Torstein Honsi,
I also required same functionality.
I want to export multiple chart on single pdf but on different pages. can you please suggest any solution for this.

**Comment by TorsteinHonsi** [comment]
Any idea @cvasseng @larsac07 ?

**Comment by larsac07** [comment]
The solution is not as complicated in itself. By –
1. Sending an SVG with multiple `<g>`'s containing each chart, like in the example used by @mayurtanna
2. Re-separating them into different `<svg>`'s in the server
3. Setting [`paperSize` and adding a `<div style="page-break-before:always;"></div>`](http://stackoverflow.com/questions/17027142/phantomjs-how-to-render-a-multi-page-pdf) between each `<g>`

– you'll get the same result.

But:
- It will require a larger restructuring of [`worker.js`](https://github.com/highcharts/node-export-server/blob/master/phantom/worker.js)
- It will mess up rate limiting and hog lots of additional resources
- It will require running a DOMParser, which costs a lot of extra performance for each time an SVG is received (to extract the `<g>`'s. RegEx unacceptable)
- It is not really inside the scope of the export server

I'd recommend [installing your own export server](https://www.highcharts.com/docs/export-module/setting-up-the-server) and configuring it to do what you require.

**Comment by TorsteinHonsi** [comment]
It may also be possible to do something similar with the [client side PDF export](https://www.highcharts.com/docs/export-module/client-side-export), but it will also require modifications.

**Comment by binarudani** [comment]
Hi Torstein Honsi,

Thanks for reply, I have used client side PDF export initially but its not supporting Chinese characters so i have to use server side exporting.

check below link : 
https://forum.highcharts.com/highcharts-usage/highchart-offline-exporting-t37925/

**Comment by taurus49** [comment]
Hi everybody,

i meet some difficulties to export multiples charts with my own export server. How to configure it for having PDF A4 format and line break beetween each chart and send a post request with the svg of charts to my server ?

**Comment by pawelfus** [comment]
As said above: 

> Our export server is made for creating one page per SVG

If you use first solution from FAQ:  https://www.highcharts.com/docs/getting-started/frequently-asked-questions#export-multiple - then charts are rendered in one SVG. 

Note: This ticket is still open.

**Comment by taurus49** [comment]
Yeah I understand so i didn't find any documentation or tutorial showing me how to proceed to realize an export on multiple pages ? can i do it with installing my [own export server](https://www.highcharts.com/docs/export-module/setting-up-the-server) or should i test an other solution and which one please ?

**Comment by taurus49** [comment]
So i installed my own exprot server and configured the export.js file available in highcharts export server node module (npm//node_modules/highcharts-export-server/phantom/export.js) but it seems that the methode downloadSVGLocal do not use my server while i modify the highcharts url option.. So what is wrong ?

**Comment by pawelfus** [comment]
Hi @taurus49 

My apologies for the delay! Ticket is still open with "Enhancement" flag, which means we need to create a solution (thus no documentation/tutorial) for this case.

**Comment by taurus49** [comment]
Hi @pawelfus,

no problem, i hope there is a solution to produce reports with a multiple page formatter containing each a unique chart. 

Thanks for your reply !

**Comment by Cyril25** [comment]
Hi everybody, 
I also required the same functionality.
Thanks.

**Comment by taurus49** [comment]
Hi,

do anyone have an answer ?

**Comment by sebastianbochan** [comment]
The ticket has still enhancement status which means that the feature is not available yet.

**Comment by ynixt** [comment]
This functionality would be very useful

**Comment by waquasahmad** [comment]
Hi ,
I have a requirement for generating multipage PDF with charts and text . Its already working using PhantomJS highchatrt converts module . But we want to use node JS export server module .
Is this possible using the node js export server module

**Comment by TorsteinHonsi** [comment]
@cvasseng

**Comment by SAGARKanuru** [comment]
Hi,
if any one have fiddle of this solution. Could you please add comment here. That will help. Thanks.

**Comment by taylorjboone** [comment]
Good morning/afternoon, is the reopening of this ticket an indication that it is being looked at again? I've been trying to figure out a way to get multi-page exports working and haven't had much success yet.
